contents/handbook/small-teams/growth/index.mdx

```diff
@@ -15,10 +15,19 @@ Maximize the number of people who get value out of PostHog, and help them realiz
 
 ## What the Growth team does
 
-- Sign up
-- Activation and reducing time to value
-- Billing and pricing changes
-- Product led growth, including messaging, emailing and notifications
+- Anything activation
+  - Activation to features or across features, but only for existing features.
+  - Onboarding flows (global and per feature)
+  - Sign up and authentication
```

Comment:
Activation is a particular event that only happens after signup/acquisition. So I'd leave sign up and authentication as its own top-level thing.